comps/agent/langchain/src/utils.py

```diff
@@ -122,6 +123,23 @@ def has_multi_tool_inputs(tools):
     return ret
 
 
+def load_python_prompt(file_dir_path: str):
```

Comment:
What is this function used for?

Reply:
This function is used for loading modules from python file.

Reply:
I don't see this function used anywhere else?